projects/control-service/projects/pipelines_control_service/src/main/resources/schema.graphqls

```diff
@@ -17,6 +18,17 @@ enum Direction {
     DESC
 }
 
+input DataJobExecutionFilter {
+    startTime_gte: String,
```

Comment:
If you need `gte`, you would then `lte` (less than). Searching for between two periods is pretty normal. 
 
Actually for pagination might be the only way

Reply:
Pagination has not been added. Do you think we need one for this particular use case?

Reply:
Pagination needs depend on the number of items the API could return. If it is potentially a high number (maybe over 100 in our case though it depends on the size of each record) then pagination would be necessary. 

Using page_size and page_number might be the normal way to paginate.

 I just think if you need GTE , user would need less (LTE). So if the cost is minimal (e.g 10% to add both) , it might be better to add both now. But I am ok either way.

Reply:
Ok, thanks.